extensions/src/AWSSDK.Extensions.Configuration.AWSSystemsManager/AWSSDK.Extensions.Configuration.AWSSystemsManager.nuspec

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is there a reason to use nusepc file instead  of creating a package from .csproj?  Or, were you just following the convention of other SDK extension libraries?

Reply:
convention from existing SDK Projects